### PR TITLE
add more docs

### PR DESCRIPTION
**Pull request by Valexr**

### CODE REVIEW

File: src/lib/layouts/Aside/Aside.svelte [hunk 1]
```diff
@@ -119,8 +113,8 @@
 				break;
 		}
 	};
-	let ww = 0,
-		sidebar = null;
+	let ww = 0;
```
Comment (blokhin) [inline src/lib/layouts/Aside/Aside.svelte:116]: @Valexr please, let's use more meaningful name for the variable, `ww` is very bad name

File: src/routes/__layout.svelte [hunk 2]
```diff
@@ -43,10 +43,10 @@
 
 		<nav id="sidebar" slot="sidebarLeft" class="m-2">
 			<h5><a href={`${base}/`} on:click={() => (openLeft = false)}>Svelte-spectre</a></h5>
+			<br />
```
Comment (blokhin) [inline src/routes/__layout.svelte:46]: Isn't it a bad practice to use line breaks? May be CSS is better...

File: src/routes/__layout.svelte [hunk 3]
```diff
@@ -126,6 +142,7 @@
 		const mds = await Promise.all(body);
 		const links = mds.reduce((a, c) => {
```
Comment (blokhin) [inline src/routes/__layout.svelte:143]: `a` and `c` are very bad variable names

File: src/routes/docs/components/button.md [hunk 4]
```diff
@@ -106,7 +106,7 @@ config:
                     },
                 shape: { options: ['square', 'circle'] },
             },
-        States: { state: { options: ['active', 'disabled', 'loading'] } },
+        States: { state: { options: [null, 'active', 'disabled', 'loading'] } },
```
Comment (blokhin) [inline src/routes/docs/components/button.md:109]: Is `null` a valid option? :thinking:

**Review by blokhin — CHANGES_REQUESTED** [review]

File: src/routes/docs/components/button.md (hunk 4 above)
Comment (Valexr) [reply src/routes/docs/components/button.md:109]: yes

File: src/routes/__layout.svelte (hunk 3 above)
Comment (Valexr) [reply src/routes/__layout.svelte:143]: fixed

File: src/routes/__layout.svelte (hunk 2 above)
Comment (Valexr) [reply src/routes/__layout.svelte:46]: it's normal practice, but fixed in css

File: src/lib/layouts/Aside/Aside.svelte (hunk 1 above)
Comment (Valexr) [reply src/lib/layouts/Aside/Aside.svelte:116]: fixed

**Review by blokhin — APPROVED** [review]

**Pull request merged**